docs/GettingStarted.md

```diff
@@ -2024,16 +2018,30 @@ You can also set this priority manually to either drop a non-interesting trace o
 
 When not using [distributed tracing](#distributed-tracing), you may change the priority at any time, as long as the trace incomplete. But it has to be done before any context propagation (fork, RPC calls) to be useful in a distributed context. Changing the priority after the context has been propagated causes different parts of a distributed trace to use different priorities. Some parts might be kept, some parts might be rejected, and this can cause the trace to be partially stored and remain incomplete.
```

Comment:
Noticed a tiny language bug:
```suggestion
When not using [distributed tracing](#distributed-tracing), you may change the priority at any time, as long as the trace is incomplete. But it has to be done before any context propagation (fork, RPC calls) to be useful in a distributed context. Changing the priority after the context has been propagated causes different parts of a distributed trace to use different priorities. Some parts might be kept, some parts might be rejected, and this can cause the trace to be partially stored and remain incomplete.
```